functions/exercise2/scriptSpec.js

```diff
@@ -1 +1,25 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals checkPalindrome*/
+
+describe("Checking if word is palindrome: ", function(){
+    it("word tenet is a palindrome", function(){
+        var wordFromTheRear = checkPalindrome("tenet");
+        expect(wordFromTheRear).toBeTruthy();
+    });
+    it("word kayak is a palindrome", function(){
```

Comment:
Test description does not reflect what this test does.
Also this test is virtually a duplication of first test, so it is not needed and it is better to remove it.